app/components/Map.js

```diff
@@ -47,24 +47,15 @@ export default class Map extends Component {
     });
   }
```

Comment:
@asu91 t I noticed in the state there is an unnecessary state.position
and you set region = this.state.position from many PRs ago. Since we are no longer using it,
on lines 188-189 can we actually change the props on mapview component to be : 

```javascript
initialRegion={stateLocation}
region={stateLocation}
```
- this creates a smoother transition to the map.